<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoAtlasTest</class>
 <widget class="QWidget" name="DemoAtlasTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <item>
      <widget class="PwGisWidget" name="pwgiswidget" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>170</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnOpenMap">
           <property name="text">
            <string>Open Map</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCloseMap">
           <property name="text">
            <string>Close Map</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOpenAtlas">
           <property name="text">
            <string>Open Atlas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCloseAtlas">
           <property name="text">
            <string>Close Atlas</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>65</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Signal Counters</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>399</width>
            <height>48</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="mapReadyLabel">
             <property name="text">
              <string>mapReady:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="atlasReadyEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="atlasReadyLabel">
             <property name="text">
              <string>atlasReady:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="mapReadyEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_mF">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>20</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>openMapFrom</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_aF">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>40</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>openAtlasFrom</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
